float32[] state
float32[] action
---
float32[] next_state_mean
float32[] next_state_std

